pkg/scheduler/flavorassigner/flavorassigner.go

```diff
@@ -573,6 +573,7 @@ func fitsResourceQuota(fName kueue.ResourceFlavorReference, rName corev1.Resourc
 	cohortUsed := used
 	if cq.Cohort != nil {
 		cohortUsed = cq.Cohort.Usage[fName][rName]
+		cohortAvailable = cq.GetRequestableQuota(fName, rName)
```

Comment:
Here is to recalculate cohort.RequestableResources when LendingLimit enabled.
I agree it seems a little confusing, I can add some comments here. WDYT?

Reply:
Done.

Reply:
I just moved this to L557, because LendingLimit may influence to the BorrowingLimit too.
Here is the change: [commit](https://github.com/kubernetes-sigs/kueue/pull/1385/commits/121e65d9ca8548c9d88bf39bb65e9ea5736b54c3).

Reply:
@B1F030 Did you push the commit? I could not see the commit.

Reply:
> I just moved this to L557, because LendingLimit may influence to the BorrowingLimit too.
Here is the change: [commit](https://github.com/kubernetes-sigs/kueue/pull/1385/commits/121e65d9ca8548c9d88bf39bb65e9ea5736b54c3).

Oh, what a timing. I could see the commit. Let me check 👀